Makefile

```diff
@@ -103,6 +104,14 @@ else
   FIND_COMMAND := find -E $(GIT_EXPORT_DIR)
 endif
 
+$(VENV): bin/require.pip
+	virtualenv -p python2.7 $(VENV)
+	. $(VENV)/bin/activate && pip install -r bin/require.pip
```

Comment:
> This could be $(VENV)/bin/pip install -r bin/require.pip

Actually, that doesn't seem to work for pip install - looks like it needs the activation first.